config/action.d/cloudflare.conf

```diff
@@ -43,13 +48,18 @@ actionban = curl https://www.cloudflare.com/api_json.html -d 'a=ban' -d 'tkn=<cf
 #          <time>  unix timestamp of the ban time
 # Values:  CMD
 #
-actionunban = curl https://www.cloudflare.com/api_json.html -d 'a=nul' -d 'tkn=<cftoken>' -d 'email=<cfuser>' -d 'key=<ip>'
-
+actionunban = curl -s -o /dev/null https://www.cloudflare.com/api_json.html -d 'a=nul' -d 'tkn=<cftoken>' -d 'email=<cfuser>' -d 'key=<ip>'
 
 [Init]
 
-# Default Cloudflare API token 
-cftoken = 
+# This action needs to be set up in jail.local or in jail.d/somefile.conf
+#
+# action = %(action_cf_mwl)s
```

Comment:
I know that you have defined it!  But I meant that people do not have to use this composite action, and it is better to document here how THIS action alone could be used (although it is somewhat obvious and other actions do not dive into this gory descriptions per each action).  But ok -- for this one, then expand

```This action needs to be set up in jail.local or in jail.d/somefile.conf```
into
```If you like to use this action with mailing whois lines, you could use predefined in jail.conf
    action_cf_mwl composite action, just define in your jail:```

smth like that...